files/base.rb

```diff
@@ -229,6 +229,18 @@ def filter_repeated
 
     initial_failing_occurrences = interval > 0 ? (alert_after / interval) : 0
     number_of_failed_attempts = @event['occurrences'] - initial_failing_occurrences
+    # sensu 0.26+ only, else never make  occurrences_watermark actionable
+    occurrences_watermark = @event.key?('occurrences_watermark') ? event['occurrences_watermark'] : -1
```

Comment:
I would put all of your logic into ```if @event.key?('occurrences_watermark')``` instead of assigning it a phony value of -1.